Makefile

```
@@ -31,6 +31,11 @@ OCM_CLIENT_SECRET := ${OCM_CLIENT_SECRET}
 ENABLE_AUTH := $(or ${ENABLE_AUTH},False)
 DELETE_PVC := $(or ${DELETE_PVC},False)
 
+ifneq ($(or $(INSTALLATION_TIMEOUT),),)
+        INSTALLATION_TIMEOUT_FLAG = --installation-timeout $(INSTALLATION_TIMEOUT)
+endif
```

Unsure why you need `($(or $(INSTALLATION_TIMEOUT),),)`?
```suggestion
ifdef INSTALLATION_TIMEOUT
        INSTALLATION_TIMEOUT_FLAG = --installation-timeout $(INSTALLATION_TIMEOUT)
endif
```

---

Consider moving it to the deploy section where it's used